src/auth/auth.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { JwtModule } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { JwtStrategy } from './jwt.strategy';
+import {UsersModule} from "../users/user.module";
```

Comment:
[minor] formatting issue

It's concerning that I'm seeing some formatting issues which means you either don't have a plugin installed or it's not working right.